handlers/ui_functions.py

```diff
@@ -0,0 +1,48 @@
+import os
+import sys
+from datetime import datetime
+from handlers import driver
+from handlers.gdrive_login_page import GDriveLoginUI
+from handlers.gdrive_logout_page import GDriveLogoutUI
+from handlers.gdrive_page import GDriveUI
+from pathlib import Path
+from time import sleep
+
+credentials_path = '../credentials/login_credentials.txt'
```

Comment:
[Comment for the future]

it's better to remove all of the hard-coded paths and move those things to env variables. Though, let's leave it for now. Please fix review comments (non-optional ones) and we can merge this branch to master.